project/src/components/sorting/sorting.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { useAppDispatch } from '../../hooks';
+import { SORTING_OPTIONS } from '../../const';
+import { changeSorting } from '../../store/action';
+
+type SortingProps = {
+  selectedSorting: string;
+}
+
+function Sorting({selectedSorting}: SortingProps): JSX.Element {
+  const [isOpenSorting, setOpenSorting] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isOpenSorting, setOpenSorting] = useState(false);
```
Если ты сразу задаешь значение, то тс подставит его тип автоматически